atomic_reactor/utils/operator.py

```diff
@@ -200,6 +199,19 @@ def check_csv(data, schema):
     validate_with_schema(data, schema)
 
 
+def modify_dict_recursively(target, mods, append=False):
+    for key, value in mods.items():
+        if isinstance(value, dict):
+            # recursive call
+            target[key] = modify_dict_recursively(target.get(key, {}), value, append=append)
+        else:
+            if append:
+                target.setdefault(key, []).extend(value)
+            else:  # Got `None`, create key & value
```

Comment:
Why `None` ?

Reply:
Ah, that's a leftover from something else